packages/site/cms-site/src/blocks/PixelImageBlock.module.scss

```diff
@@ -0,0 +1,6 @@
+.imageContainer {
+    position: relative;
+    width: 100%;
+    height: 100%;
+    aspect-ratio: var(--aspect-ratio);
```

Comment:
@jamesricky could it be a problem if both DAM video block and pixel image block use `--aspect-ratio`? Should we add a prefix?

Reply:
CSS variables follow the same cascading rules as other css defintions. Meaning a CSS variable defined on an element via style only affects this element and it's children. So PixelImageBlock and DamVideoBlock won't affect each other

Reply:
DamVideoBlock renders a PixelImageBlock as its preview image, so in theory the cascading is given. But since PixelImageBlock always defines --aspect-ratio (which DamVideoBlock doesn't), it shouldn't be a problem.

Reply:
Yes, in this case the PixelImageBlock aspect ratio should override the DamVideoBlock aspect ratio. Also, the aspectRatio is passed from DamVideoBlock to the preview image. So it should always be the same anyway: https://github.com/vivid-planet/comet/blob/da9f4d67807e43380bceaeff9a26afa21988155f/packages/site/cms-site/src/blocks/DamVideoBlock.tsx#L62

In theory there could be scenarios where variables with the same name could cause problems. But I don't think it happens anywhere in the library ATM